python/usb_console/simulation.py

```diff
@@ -18,11 +23,14 @@ def __init__(self,devices,seed):
         """
         self.devices = devices
         self.flight_controller = self.devices['FlightController']
+        self.truth_trajectory = []
+        self.log = ""
 
     def start(self, duration):
         self.sim_thread = threading.Thread(name="Python-MATLAB Simulation Interface",
                                            target=self.run,
                                            args=[duration])
+        self.running = True
         self.sim_thread.start()
 
     def run(self,duration):
```

Comment:
This function essentially does the same thing as main.m in the MATLAB/ folder. It has to, in order to run the simulation.